src/components/QuizCreator.vue

```diff
@@ -42,7 +42,7 @@ import { handleSubmitData } from "../dataAccessLayer.ts";
 import { base64ToFile } from "../utils";
 
 const quizType = ref("Image");
-const imageSrc = ref<string | undefined>();
+const imageSrc = ref<string | undefined>("");
```

Comment:
we can simplify to one string type, since default value provided?

Reply:
sure we can just have as string